woocommerce/woocommerce-ios
# Tool: Wormholy integration into the debug build

Fixes #1502 

## Description
This PR integrates [Wormholy](https://github.com/pmusolino/Wormholy) as a network debugging library inside Woo App. I have integrated it only on the `Debug` configuration, and it's launched programmatically from settings. So if you want to use it, you need to go into app settings.

## Testing
1) Launch the app with the `Debug` configuration on your iOS simulator or device.
2) Go to Settings
3) Under the section `Other` you will able to see a new cell **Launch Wormholy debug**
4) Tap it
5) Wormholy will be presented modally.
6) Make sure CI pass

## Screencast
![wormholy integration in woo app](https://user-images.githubusercontent.com/495617/69878967-8c9ac800-12c6-11ea-99f1-dfc3e8ffc057.gif)


Update release notes:
